CHANGELOG.md

```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0-beta.3
+
+_Feb 16, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Charts get a [built in grid](https://next.mui.com/x/react-charts/axis/#grid)
+  ![image](https://github.com/mui/mui-x/assets/45398769/74299f54-f020-4135-b38c-dc88a230db30)
+- 🐞 Bugfixes
```

Comment:
```suggestion
- 🎛️ Charts get a [Gauge component](https://next.mui.com/x/react-charts/gauge/) useful to display a value in a dashboard.
  ![image](https://github.com/mui/mui-x/assets/45398769/fb7a94b5-bef6-4fc2-a0cd-d6ff5b60fa8b)
- 🐞 Bugfixes
```

Reply:
Would it also make sense to highlight the addition of `ChartsGrid`? 🤔

Reply:
Isn't it the same as the first highlight?

Reply:
🤦 
Forget what I said... The diff confused me...